BUILDING.md

```diff
@@ -9,15 +9,24 @@ If you consistently can reproduce a test failure, search for it in the
 file a new issue.
 
 
-### Unix / Macintosh
+### Unix / OS X
 
 Prerequisites:
 
 * `gcc` and `g++` 4.8 or newer, or
 * `clang` and `clang++` 3.4 or newer
 * Python 2.6 or 2.7
 * GNU Make 3.81 or newer
-* libexecinfo (FreeBSD and OpenBSD only)
+
+On OS X, you will also need:
+* [Xcode](https://developer.apple.com/xcode/download/)
+  * You also need to install the `Command Line Tools` via Xcode. You can find
+    this under the menu `Xcode -> Preferences -> Downloads`
+  * This step will install `gcc` and the related toolchain containing `make`
+
+On FreeBSD and OpenBSD, you will also need:
+* libexecinfo
```

Comment:
I actually don't think you need this any longer on FreeBSD. The only dependencies I had to install was: `pkg install python gmake` (python installs python2.7 and python which symlinks to python2).

Can't speak for OpenBSD though.